src/redux/reducers/authReducer.js

```diff
@@ -0,0 +1,51 @@
+import { pending, fulfilled, rejected, baseState } from 'redux/utils';
+import { LOGIN_USER, SET_AUTH_USER } from 'redux/actions/actionTypes';
+
+const initialState = baseState('user', {});
+const authState = {
+  loggedIn: false,
+  user: {
+    first_name: 'Shen',
```

Comment:
Can we remove these dummy data from this lady

Reply:
Yeah, sure. I gonna remove them after finishing the auth feature